### PR TITLE
doc: sparse vector

### PR DESCRIPTION
**Pull request by silver-ymz**
- feature PR: https://github.com/tensorchord/pgvecto.rs/pull/299

### CODE REVIEW

File: src/reference/schema.md [hunk 1]
```diff
@@ -4,48 +4,68 @@ Here is the schema provided by `pgvecto.rs`.
...
+| <#>  | svector       | svector        | real        | negative dot product        |
 | <=>  | vector        | vector         | real        | cosine distance             |
 | <=>  | vecf16        | vecf16         | real        | cosine distance             |
+| <=>  | svector       | svector        | real        | cosine distance             |
 
 ## List of functions
 
+| Name                     | Argument type                               | Result type | Description                                                                         |
+| ------------------------ | ------------------------------------------- | ----------- | ----------------------------------------------------------------------------------- |
+| svector_from_kv_string   | dims integer, input text                    | svector     | construct a sparse vector from a string of key-value pairs, e.x. `[{0: 1}, {1: 1}]` |
```
Comment (usamoi) [inline src/reference/schema.md:56]: Please correct them.

Comment (silver-ymz) [reply src/reference/schema.md]: updated in new commit

File: src/reference/schema.md [hunk 2]
```diff
@@ -4,48 +4,67 @@ Here is the schema provided by `pgvecto.rs`.
...
+| <#>  | svector       | svector        | real        | negative dot product        |
 | <=>  | vector        | vector         | real        | cosine distance             |
 | <=>  | vecf16        | vecf16         | real        | cosine distance             |
+| <=>  | svector       | svector        | real        | cosine distance             |
 
 ## List of functions
 
+| Name                     | Argument type                               | Result type | Description                                                     |
+| ------------------------ | ------------------------------------------- | ----------- | --------------------------------------------------------------- |
+| svector_from_split_array | dims integer, index integer[], value real[] | svector     | construct a sparse vector from two arrays of indexes and values |
```
Comment (usamoi) [inline src/reference/schema.md:56]: The name should be `to_svector`.

Comment (silver-ymz) [reply src/reference/schema.md]: updated in new commit

**Review by usamoi — APPROVED** [review]

**Pull request merged**